### PR TITLE
Replicate

### PR DESCRIPTION
**Pull request by adampingel**
- Use public models
- Mention credit link
- Simplify, but still maintain blocking and async examples.

### CODE REVIEW

File: recipes/Getting_Started_with_Granite_Code.ipynb [hunk 1]
```diff
@@ -15,29 +15,39 @@
    "source": [
     "## Introduction\n",
     "\n",
-    "This notebook demonstrates using inference calls against a model hosted on [Replicate](https://replicate.com/). To see how you can use [Ollama](https://ollama.com/) to host models locally instead, see the [Coding Assistant in VSCode](Coding_Assistant_in_VSCode/Coding_Assistant_in_VSCode.ipynb) recipe."
+    "This notebook demonstrates using inference calls against a model hosted on [Replicate](https://replicate.com/).  To see how you can use [Ollama](https://ollama.com/) to host models locally instead, see the [Continue VSCode](Continue_VSCode/Continue_VSCode.ipynb) recipe.\n",
```
Comment (deanwampler) [inline recipes/Getting_Started_with_Granite_Code.ipynb:18]: We're planning to refactor out the Ollama instructions into a separate notebook, right? I definitely think we should.

File: recipes/Getting_Started_with_Granite_Code.ipynb [hunk 2]
```diff
@@ -101,172 +108,87 @@
...
-      "Requirement already satisfied: idna in /opt/homebrew/Caskroom/miniforge/base/envs/granite-code/lib/python3.11/site-packages (from httpx<1,>=0.21.0->replicate) (3.7)\n",
-      "Requirement already satisfied: sniffio in /opt/homebrew/Caskroom/miniforge/base/envs/granite-code/lib/python3.11/site-packages (from httpx<1,>=0.21.0->replicate) (1.3.0)\n",
-      "Requirement already satisfied: h11<0.15,>=0.13 in /opt/homebrew/Caskroom/miniforge/base/envs/granite-code/lib/python3.11/site-packages (from httpcore==1.*->httpx<1,>=0.21.0->replicate) (0.14.0)\n",
-      "Requirement already satisfied: annotated-types>=0.4.0 in /opt/homebrew/Caskroom/miniforge/base/envs/granite-code/lib/python3.11/site-packages (from pydantic>1.10.7->replicate) (0.7.0)\n",
-      "Requirement already satisfied: pydantic-core==2.18.2 in /opt/homebrew/Caskroom/miniforge/base/envs/granite-code/lib/python3.11/site-packages (from pydantic>1.10.7->replicate) (2.18.2)\n",
-      "Note: you may need to restart the kernel to use updated packages.\n"
+      "Sure, here's an example SQL query that fetches all columns for the first 50 rows in a table named 'users':\n",
```
Comment (deanwampler) [inline recipes/Getting_Started_with_Granite_Code.ipynb:127]: Do you want to clear the output?

**Review by deanwampler — APPROVED** [review]
I made a few suggestions, but LGTM otherwise.

File: recipes/Getting_Started_with_Granite_Code.ipynb [hunk 3]
```diff
@@ -5,7 +5,7 @@
    "id": "3e98b2a4-cf47-4523-bd07-827683728d31",
    "metadata": {},
    "source": [
-    "# Getting Started with Granite Code"
+    "# Use Granite Code Hosted on Replicate"
```
[the remark this answers is not in the document]
Comment (adampingel) [reply recipes/Getting_Started_with_Granite_Code.ipynb:8]: I agree, but we need to coordinate that with the site referencing the notebook.  Let's take that as a separate task -- there is other renaming I'd like to do.

**Pull request merged**